todo_app/data/mongodb_items.py

```diff
@@ -0,0 +1,37 @@
+from todo_app.flask_config import Config
+from todo_app.data.items import Item
+import pymongo
+
+class MongoDBItems:
+    def __init__(self):
+        self.config = Config()
+        self.client = pymongo.MongoClient(self.config.MONGODB_CONNECTION_STRING)
+        self.db = self.client[self.config.MONGODB_DB_NAME]
+        self.collection = self.db[self.config.MONGODB_COLLECTION_NAME]
+
+    def get_items(self):
+        mongodb_documents = list(self.collection.find())
+        items = []
+        for document in mongodb_documents:
+            item = Item.from_mongodb_document(document)
+            items.append(item)
+        return items
+
+    def add_item(self, title):
+        item = {
+            "id": str(len(list(self.collection.find())) + 1),
+            "name": title,
+            "status": "To Do"
+        }
+        self.collection.insert_one(item)
+
+    def change_item_status(self, id):
+        current_status = self.collection.find_one({"id": id})["status"]
+        new_status = "Done" if current_status == "To Do" else "To Do"
```

Comment:
(Not important but) might be worth extracting the item statuses into constants (or an Enum type)